immutables/_map.c

```diff
@@ -1,4 +1,5 @@
 #include <stddef.h> /* For offsetof */
+#include "pythoncapi_compat.h"
```

Comment:
This means we need to drop support for Python 3.5, right?

Reply:
I updated https://github.com/pythoncapi/pythoncapi_compat to support Python 2.7, 3.4 and 3.5 as well.

I updated pythoncapi_compat.h to the latest version in my PR.

Reply:
Python 3.5 was already supported by I was too lazy to port the tests to Python 3.5. Since I'm trying to use pythoncapi_compat.h on Mercurial which still supports Python 2.7, I ported it to Python 2.7, 3.4 and 3.5. And 2.7 and 3.5 are now tested on the project CI.